pages/mods.vue

```diff
@@ -543,7 +543,7 @@ export default {
         const pageAmount = Math.ceil(res.data.total_hits / res.data.limit)
 
         this.currentPage = newPageNumber
-        if (pageAmount > 4) {
+        if (pageAmount > 3) {
```

Comment:
![image](https://i.imgur.com/VOcF59j.png)

The first `-` can be replaced with a 2, not sure how hard that'd be to do

Reply:
Should be all fixed